src/formatBinaryPrefixAuto.js

```diff
@@ -0,0 +1,46 @@
+export var binaryPrefixExponent;
+
+export default function(x, p) {
+  var binaryExponent = 0;
+  if (x === Infinity) return binaryPrefixExponent = 0, x;
```

Comment:
This line is confusing.

Why not

```js
if (x === Infinity) return Infinity;
```

?

Reply:
Presumably because it needs to set the exported binaryPrefixExponent (note, not binaryExponent).

Reply:
Aaaahhhhh I see. Side effects. Thanks for pointing that out 🙏